### PR TITLE
Add gots.history source

### PR DESCRIPTION
**Pull request by Yuri364**
### Added

- Source `gots.history`
- Field with path `gots_auctions.items[].vehicle.mileage`
- Field with path `gots_auctions.items[].vehicle.damage.value`
- Field with path `gots_auctions.items[].vehicle.damage.descriptions`
- Field with path `gots_auctions.items[].auction.date`
- Field with path `gots_auctions.items[].auction.number`
- Field with path `gots_auctions.items[].auction.client.name`
- Field with path `gots_auctions.items[].auction.client.postal_code`
- Field with path `gots_auctions.items[].auction.client.city`
- Field with path `gots_auctions.items[].auction.client.address_line`
- Field with path `gots_auctions.items[].auction.client.phone_number`

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
...
+- Field with path `gots_auctions.items[].vehicle.mileage`
+- Field with path `gots_auctions.items[].vehicle.damage.value`
+- Field with path `gots_auctions.items[].vehicle.damage.descriptions`
+- Field with path `gots_auctions.items[].auction.date`
+- Field with path `gots_auctions.items[].auction.number`
+- Field with path `gots_auctions.items[].auction.client.name`
+- Field with path `gots_auctions.items[].auction.client.postal_code`
+- Field with path `gots_auctions.items[].auction.client.city`
+- Field with path `gots_auctions.items[].auction.client.address_line`
+- Field with path `gots_auctions.items[].auction.client.phone_number`
```
Comment (Zaboday) [inline CHANGELOG.md:21]: Need to add more fields:

- Field with path `gots_auctions.items[].vehicle.damage.descriptions[].code`
- Field with path `gots_auctions.items[].vehicle.damage.descriptions[].value`
- Field with path `gots_auctions.count`

File: fields/default/fields_list.json [hunk 2]
```diff
@@ -4336,6 +4336,106 @@
...
+        "description": "Размещение ТС на аукционах ГОТС: Значение повреждения ТС",
+        "types": [
+            "float"
+        ],
+        "fillable_by": [
+            "gots.history"
+        ]
+    },
+    {
+        "path": "gots_auctions.items[].vehicle.damage.descriptions",
```
Comment (Zaboday) [inline fields/default/fields_list.json:4360]: Unwrap this fild with:
```
{
        "path": "gots_auctions.items[].vehicle.damage.descriptions[].code",
        "description": "Размещение ТС на аукционах ГОТС: Описание повреждений ТС, код",
        "types": [
            "string"
        ],
        "fillable_by": [
            "gots.history"
        ]
    }
```

```
{
        "path": "gots_auctions.items[].vehicle.damage.descriptions[].value",
        "description": "Размещение ТС на аукционах ГОТС: Описание повреждений ТС",
        "types": [
            "string"
        ],
        "fillable_by": [
            "gots.history"
        ]
    }
```

File: fields/default/fields_list.json [hunk 3]
```diff
@@ -4336,6 +4336,106 @@
...
+        "types": [
+            "string"
+        ],
+        "fillable_by": [
+            "gots.history"
+        ]
+    },
+    {
+        "path": "gots_auctions.items[].auction.client.postal_code",
+        "description": "Размещение ТС на аукционах ГОТС: Индекс клиента разместившего лот",
```
Comment (Zaboday) [inline fields/default/fields_list.json:4401]: `Индекс клиента...` -> `Почтовый индекс клиента...`

File: reports/default/json-schema.json [hunk 4]
```diff
@@ -3056,6 +3056,250 @@
...
+                                                "examples": [
+                                                    20562.5
+                                                ],
+                                                "fillable_by": [
+                                                    "gots.history"
+                                                ]
+                                            },
+                                            "descriptions": {
+                                                "description": "Описание повреждений ТС",
+                                                "oneOf": [
```
Comment (Zaboday) [inline reports/default/json-schema.json:3106]: Should be only  `"type": "array"`.

File: fields/default/fields_list.json (hunk 2 above)
Comment (Yuri364) [reply fields/default/fields_list.json]: Added

File: fields/default/fields_list.json (hunk 3 above)
Comment (Yuri364) [reply fields/default/fields_list.json]: Fix

File: reports/default/json-schema.json (hunk 4 above)
Comment (Yuri364) [reply reports/default/json-schema.json]: Removed `oneOf`

**Review by Zaboday — APPROVED** [review]

**Pull request merged**